setup.cfg

```diff
@@ -8,7 +8,7 @@ line_length = 79
 multi_line_output = 0
 known_standard_library = setuptools
 known_first_party = mmseg
-known_third_party = PIL,cityscapesscripts,cv2,detail,matplotlib,mmcv,numpy,onnxruntime,packaging,prettytable,pytest,pytorch_sphinx_theme,requests,scipy,seaborn,torch,ts
+known_third_party = PIL,cityscapesscripts,cv2,detail,einops,matplotlib,mmcv,numpy,onnxruntime,packaging,prettytable,pytest,pytorch_sphinx_theme,requests,scipy,seaborn,torch,ts
```

Comment:
delete `einops`

Reply:
could you propose a change to the einops operations? I am used to einops and you may know better torch equivalent operations to split and permute tensors.

Reply:
done